### DISCUSSION TITLE
Adjust / balance spacings after the little "i"

### DISCUSSION THREAD

**Discussion by Ferdinand-Weinrother** [coreyhu/Urbanist#33] [Ideas]
Hello. You built a really great font - it is especially gorgeous at thin font weights. Really nice.
I got a little change request. If you look at the small "i", I got the feeling, that there is definitely too much space next to another letter at weights more than 500. 400 is fine. What do you think?

![Bildschirmfoto 2021-08-23 um 21 49 41](https://user-images.githubusercontent.com/61787159/130510263-92489baf-fc7f-4bd8-a8f4-9660bc4dcc3f.png)
![Bildschirmfoto 2021-08-23 um 21 49 55](https://user-images.githubusercontent.com/61787159/130510282-e5d89df2-6ba3-4f3f-926c-0c3c009ad1b4.png)

**Comment by coreyhu** [comment]
Thanks for bringing this up, the kerning definitely looks off here. I'll have it fixed asap

**Reply by jamieskella** [reply]
I came here to point out the same thing! Looking forward to the fix. 

![urbanist-kerning](https://user-images.githubusercontent.com/7778197/130729863-5f9b0750-e3ba-465f-8e49-42f05ac17f58.png)

**Comment by coreyhu** [comment]
@Ferdinand-Weinrother @jamieskella Just merged PR #34 for Urbanist 1.253 that should fix this ([download the build from the releases page](https://github.com/coreyhu/Urbanist/releases/tag/1.253)). Please let me know if that helps! Thanks

**Reply by jamieskella** [reply]
At glance, this looks like it has addressed the issue for me - thanks! 

When will this roll out to Google Fonts? I'm using Extensis Fonts to render Urbanist in Google Slides, so it would be fantastic to see the fix in place there as well.

**Reply by coreyhu** [reply]
It will be updated on Google Fonts, but I don't have any control over how long it may take for the update to go live. I expect maybe a few weeks at best. Sorry about any inconvenience, I'll do my best to speed things along.
